{"name": "Получение списка заказов с несуществующим courierId", "status": "passed", "description": "Проверка, что при передаче несуществующего courierId возвращается ошибка 404", "steps": [{"name": "Отправка GET-запроса на получение списка заказов с несуществующим courierId=1000000", "status": "passed", "start": 1725830829162, "stop": 1725830829464}, {"name": "Проверка, что статус код равен 404", "status": "passed", "start": 1725830829464, "stop": 1725830829464}, {"name": "Проверка, что в теле ответа присутствует сообщение об ошибке", "status": "passed", "start": 1725830829464, "stop": 1725830829464}], "start": 1725830829162, "stop": 1725830829465, "uuid": "df565a73-5b36-4c19-b741-e17bbcd5ab04", "historyId": "5916cafc1a4b51f250dd790579e46ced", "testCaseId": "5916cafc1a4b51f250dd790579e46ced", "fullName": "tests.test_orders#test_get_orders_with_invalid_courier_id", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_orders"}, {"name": "host", "value": "Asus.local"}, {"name": "thread", "value": "54554-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_orders"}]}